scripts/audit-fix-sample-modules.js

```diff
@@ -40,9 +40,7 @@ const runBatchedModuleCommand = async (command) => {
   const moduleVersionPaths = await getModuleVersionPaths();
   const batches = basicBatch(moduleVersionPaths);
 
-  for (let batchNumber = 0; batchNumber < batches.length; batchNumber += 1) {
-    const currentBatch = batches[batchNumber];
-    // eslint-disable-next-line no-await-in-loop
+  batches.forEach(async (currentBatch) => {
```

Comment:
This will make all the batches run at once rather than consecutively. You could do it with reduce.